ivadomed/config/config_new_loader.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
I am guessing we will use this file during the prototyping / development of the new loader before removing it when fully integrated. Is it the idea?